patroni/dcs/raft.py

```diff
@@ -223,50 +228,50 @@ def delete(self, key, recursive=False, **kwargs):
             return False
 
     @staticmethod
-    def __values_match(old, new):
+    def __values_match(old: Dict[str, Any], new: Dict[str, Any]) -> bool:
         return all(old.get(n) == new.get(n) for n in ('created', 'updated', 'expire', 'value'))
 
     @replicated
-    def _expire(self, key, value, callback=None):
+    def _expire(self, key: str, value: Dict[str, Any], callback: Optional[Callable[..., Any]] = None) -> None:
         current = self.__data.get(key)
         if current and self.__values_match(current, value):
             self.__pop(key)
 
-    def __expire_keys(self):
+    def __expire_keys(self) -> None:
         for key, value in self.__data.items():
             if value and 'expire' in value and value['expire'] <= time.time() and \
                     not (key in self.__limb and self.__values_match(self.__limb[key], value)):
                 self.__limb[key] = value
 
-                def callback(*args):
+                def callback(*args: Any) -> None:
                     if key in self.__limb and self.__values_match(self.__limb[key], value):
                         self.__limb.pop(key)
                 self._expire(key, value, callback=callback)
```

Comment:
```suggestion
                self._expire(key, value)
```

`callback` doesn't seem to be used for anything.

Reply:
It is intercepted and called from the function implementing `@replicated` decorator.